packages/flame-css/src/mixins/media-queries.scss

```diff
@@ -0,0 +1,52 @@
+/*
+  Media Queries
+  ======
+  A mobile first approach for responsive UI.
+*/
+
+$fl-tablet-portrait-width: 600px;
+$fl-tablet-landscape-width: 900px;
+$fl-tablet-desktop-width: 1200px;
+$fl-tablet-big-desktop-width: 1800px;
```

Comment:
Those should follow breakpoints we have in tokens: https://github.com/lightspeed/flame/blob/master/packages/flame-tokens/src/theme-ui/common.ts#L4

Reply:
removing this entire file as this is opening a can of worms that I do not want to tackle in this PR